Makefile

```diff
@@ -179,25 +184,22 @@ e2e-cleanup:
 
 image-cleanup:
 # Delete Docker image
-ifneq ($(shell docker images -q vela-core-test:$(GIT_COMMIT)),)
-	docker image rm -f vela-core-test:$(GIT_COMMIT)
+ifneq ($(shell docker images -q $(VELA_CORE_TEST_IMAGE)),)
+	docker rmi -f $(VELA_CORE_TEST_IMAGE)
 endif
 
 # load docker image to the kind cluster
 kind-load:
-	docker build -t vela-core-test:$(GIT_COMMIT) .
-	kind load docker-image vela-core-test:$(GIT_COMMIT) || { echo >&2 "kind not installed or error loading image: $(IMAGE)"; exit 1; }
-
-# Image URL to use all building/pushing image targets
-IMG ?= vela-core:latest
+	docker build -t $(VELA_CORE_TEST_IMAGE) .
+	kind load docker-image $(VELA_CORE_TEST_IMAGE) || { echo >&2 "kind not installed or error loading image: $(VELA_CORE_TEST_IMAGE)"; exit 1; }
 
 # Run tests
 core-test: fmt vet manifests
 	go test ./pkg/... -coverprofile cover.out
 
-# Build manager binary
+# Build vela core manager binary
 manager: fmt vet lint manifests
-	go build -o bin/manager ./cmd/core/main.go
+	$(GOBUILD_ENV) go build -o bin/manager -a -ldflags $(LDFLAGS) ./cmd/core/main.go
```

Comment:
does the ldflags really be used?

Reply:
The original plan was not to add `ldflags`, but vela-core has no version information. I plan to add the `manager version` subcommand, so I added it in advance

Reply:
Otherwise I have to distinguish between `-s -w` and `-X`

Reply:
Sure, go ahead, sounds very good

Reply:
maybe print the version information at startup time is enough.